userChrome.css

```diff
@@ -46,3 +46,41 @@
 #reload-button {
 	margin-block-start: -2px !important;
 }
+
+
+/* context menuitem padding and rounded hover style changes */
+ menupopup > menuitem,
+ menupopup > menu
+  {
+     border-radius: 4px !important;
```

Comment:
There's probably a solution to reduce the number of time the border-radius is set.

Reply:
Ok I'll reduce the repetitive codes